test-network-function/certification/suite.go

```diff
@@ -240,3 +262,41 @@ func GetOcpVersion() string {
 	}
 	return ocVersion
 }
+func GetKubeVersion() string {
+	ocCmd := kubernatesVersionCommand
+	kubeVersion := execCommandOutput(ocCmd)
+	if kubeVersion != outMinikubeVersion {
+		kubeVersion = strings.Split(kubeVersion, "+")[0]
+		kubeVersion = kubeVersion[1:]
+	} else {
+		kubeVersion = ""
+	}
+	return kubeVersion
+}
+func CompareNumber(num1, num2, kind string) bool {
+	newnum1 := strings.Split(num1, ".")
+	firstNum1, _ := strconv.ParseFloat(newnum1[0]+"."+newnum1[1], 64)
```

Comment:
To compare version, maybe we can use a library, for instance:

There is a nice solution from Hashicorp - https://github.com/hashicorp/go-version

import github.com/hashicorp/go-version
v1, err := version.NewVersion("1.2")
v2, err := version.NewVersion("1.5+metadata")
// Comparison example. There is also GreaterThan, Equal, and just
// a simple Compare that returns an int allowing easy >=, <=, etc.
if v1.LessThan(v2) {
    fmt.Printf("%s is less than %s", v1, v2)
}